Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ruby:2.6
+
+RUN gem install zold
+EXPOSE 4096
+
+RUN echo "#!/bin/bash" > node.sh
+RUN echo "zold node --nohup \044\100" >> node.sh
+RUN echo "tail -f zold.log" >> node.sh
+RUN chmod +x /node.sh
+
+RUN mkdir /zold
+WORKDIR /zold
+
+CMD ["/node.sh"]
```

Comment:
@pin2t can't you just do:

```
CMD ["zold node --nohup \044\100 && tail -f zold.log"]
```

Reply:
Or:

```
CMD ["/bin/bash", "-c", "zold node --nohup \044\100 && tail -f zold.log"]
```

Reply:
No. Because users have to replace CMD with their to provide an IP address of the host and invoice address (like docker run ... /node.sh --host=<your host IP> --invoice=17737fee5b825835). So if such a complex command will be used by default in the container it will be hard to start the container by users. /node.sh is just simple